corehq/apps/zapier/api/v0_5.py

```diff
@@ -23,6 +24,9 @@ def dehydrate(self, bundle):
         remove_advanced_fields(bundle.data)
         return bundle
 
+    def get_required_privilege(self):
+        return privileges.ZAPIER_INTEGRATION
```

Comment:
Does this mean a user with `API_ACCESS` will not have access to this Zapier API?

Same below.

Reply:
Correct. the permission needed to access Zapier endpoints is `ZAPIER_INTEGRATION`, not `API_ACCESS`

Reply:
I could make modifications to supports both permissions, but it doesn't seem like the intent is that these Zapier endpoints are part of our public API surface area. I interpret `API_ACCESS` to mean an external API that we're exposing to endusers. I don't think its inconsistent if this is just how our application talks to Zapier

Reply:
That sounds logically consistent to me - while it's implemented as an API, and can technically be used as one, Zapier is an integration that's priced separately from our regular APIs.